.circleci/config.yml

```diff
@@ -20,6 +20,7 @@ jobs:
       - run:
           name: Wait for containers to starts
           command: dockerize -wait http://127.0.0.1:8200 -wait tcp://127.0.0.1:3306 -timeout 30s
+      - run:
```

Comment:
Is this an accidental change?

Reply:
Oops, yeah, I was trying some stuff and missed this :)